Rocket.Chat/API/Clients/InfoClient.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  InfoClient.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 11/28/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import RealmSwift
+
+struct InfoClient: APIClient {
+    let api: AnyAPIFetcher
+
+    func fetchInfo(realm: Realm? = Realm.shared) {
+        api.fetch(InfoRequest(), succeeded: { result in
+            DispatchQueue.main.async {
+                try? realm?.write {
+                    AuthManager.isAuthenticated(realm: realm)?.serverVersion = result.version ?? ""
+                }
+            }
```

Comment:
Also moving this code (DB code) to `AuthManager`, something like `AuthManager.updateServerVersion...`